web/wp-content/themes/dctx-v2/inc/acf-gutenberg.php

```diff
@@ -0,0 +1,443 @@
+			'title'           => esc_html__( 'Call To Action', 'dctx' ),
+			'description'     => esc_html__( 'A call to action block.', 'dctx' ),
+			'render_callback' => 'dctx_acf_block_registration_callback',
+			'category'        => 'wds-blocks',
+			'icon'            => 'megaphone',
+			'keywords'        => array( 'call to action', 'cta', 'wds' ),
+			'mode'            => 'preview',
+			'enqueue_assets'  => 'dctx_acf_enqueue_backend_block_styles',
+			'align'           => 'wide',
+			'supports'        => $supports,
+			'example'         => array(
+				'attributes' => array(
+					'data' => array(
+						'title'       => esc_html__( 'Call To Action Title', 'dctx' ),
+						'text'        => esc_html__( 'Call To Action Text', 'dctx' ),
+						'button_link' => array(
+							'title' => esc_html__( 'Learn More', 'dctx' ),
+							'url'   => '#',
+						),
+					),
+				),
+			),
+		)
+	);
+
+	acf_register_block_type(
+		array(
+			'name'            => 'wds-fifty-fifty',
+			'title'           => esc_html__( 'Fifty/Fifty Block', 'dctx' ),
+			'description'     => esc_html__( 'A split-content block with text and/or media.', 'dctx' ),
+			'render_callback' => 'dctx_acf_block_registration_callback',
+			'category'        => 'wds-blocks',
+			'icon'            => 'editor-table',
+			'keywords'        => array( 'fifty fifty', 'columns', 'wds' ),
+			'mode'            => 'preview',
+			'enqueue_assets'  => 'dctx_acf_enqueue_backend_block_styles',
+			'align'           => 'wide',
+			'supports'        => $supports,
+			'example'         => array(
+				'attributes' => array(
+					'data' => array(
+						'text_primary' => esc_html__( 'Primary Text content', 'dctx' ),
+						'media_right'  => array(
+							'url' => get_stylesheet_directory_uri() . '/assets/images/placeholder.png',
+						),
+					),
+				),
+			),
+		)
+	);
+
+	acf_register_block_type(
+		array(
+			'name'            => 'wds-hero',
+			'title'           => esc_html__( 'Hero Block', 'dctx' ),
+			'description'     => esc_html__( 'A hero with an optional call to action.', 'dctx' ),
+			'render_callback' => 'dctx_acf_block_registration_callback',
+			'category'        => 'wds-blocks',
+			'icon'            => 'slides',
+			'keywords'        => array( 'hero', 'wds' ),
+			'mode'            => 'preview',
+			'enqueue_assets'  => 'dctx_acf_enqueue_backend_block_styles',
+			'align'           => 'wide',
+			'supports'        => $supports,
+			'example'         => array(
+				'attributes' => array(
+					'data' => array(
+						'title'       => esc_html__( 'Call To Action Title', 'dctx' ),
+						'text'        => esc_html__( 'Call To Action Text', 'dctx' ),
+						'button_link' => array(
+							'title' => esc_html__( 'Learn More', 'dctx' ),
+							'url'   => '#',
+						),
+						'background_options' => array(
+							'background_type' => 'color',
+							'background_color' => array(
+								'color_picker' => 'blue',
+							),
+						),
+						'display_options' => array(
```

Comment:
:warning: **Warning**: Array double arrow not aligned correctly; expected 4 space(s) between &quot;'display_options'&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).